### PR TITLE
FIT-16

### PR DESCRIPTION
**Pull request by dmitryprikhunov**
- Added Recipe Controller, Recipe Service, Recipe Repository, Recipe Entity by Pavel Khramov
- Swagger integration for APIs (some useful annotations and comments)

### CODE REVIEW

File: src/main/resources/db/master.xml [hunk 1]
```diff
@@ -3,5 +3,14 @@
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
-
+    <changeSet id="1" author="Liquibase">
```
Comment (ws-eliseev) [inline src/main/resources/db/master.xml]: Убрать

File: src/main/java/ws/eliseev/fitness/service/IRecipeService.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package ws.eliseev.fitness.service;
+
+import ws.eliseev.fitness.model.Recipe;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface IRecipeService {
+    //Вывести список всех рецептов
```
Comment (ws-eliseev) [inline src/main/java/ws/eliseev/fitness/service/IRecipeService.java]: /**

File: src/main/java/ws/eliseev/fitness/model/Recipe.java [hunk 3]
```diff
@@ -0,0 +1,75 @@
+package ws.eliseev.fitness.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.util.Objects;
+
+
```
Comment (ws-eliseev) [inline src/main/java/ws/eliseev/fitness/model/Recipe.java]: javadoc

File: src/main/java/ws/eliseev/fitness/controller/RecipeController.java [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+     * @param name - Название рецепта
+     */
+    @Operation(summary = "Gets all recipes by name", tags = "recipe")
+    @ApiResponses(value = {
+            @ApiResponse(
+                    responseCode = "200",
+                    description = "Got all recipes found by defined name"
+            )
+    })
+    @GetMapping("/recipes/name/{name}")
```
Comment (ws-eliseev) [inline src/main/java/ws/eliseev/fitness/controller/RecipeController.java]: recipes вынести наверх

**Review by ws-eliseev — APPROVED** [review]

**Review by KensynHimura — APPROVED** [review]

**Pull request merged**